apps/meteor/app/api/server/v1/rooms.ts

```diff
@@ -926,7 +926,7 @@ API.v1.addRoute(
 			const user = await Users.findOneById(this.userId, { projections: { username: 1 } });
 
 			if (!user) {
-				throw new Meteor.Error('error-invalid-user', 'Invalid user', { method: 'rooms.close' });
+				return API.v1.failure('Invalid user');
```

Comment:
This is a breaking change :/

(Not removing the Meteor error, but changing the error message. We should go with the `error-invalid-user` as it can be used to translate it on UI)

Reply:
nvm im dum